Android_Development/Java/1.6_Setup_Git_Github_On_Android_Studio.md

```diff
@@ -0,0 +1,53 @@
+# How to Setup Git And Connect Your Github Account to Android Studio.
+
+<img src="https://drive.google.com/uc?export=view&id=1hTga8r4lrNvzmtrXGBW7YzNkivSoJUMF" width="150" align = "right">

```

Comment:
#### Awesome work done @NamanManjkhola, Just a small change requested.

Make alignment in the centre and push these images on any platform other than drive because drive takes a lot of time while processing the file. (I would suggest you add these images in a separate GitHub repo).

After committing the suggested, squash your commits in a single commit.

Reply:
Aligning the images to center, right? or the whole file.